components/feature/session/src/main/java/mozilla/components/feature/session/SessionFeature.kt

```diff
@@ -56,4 +56,13 @@ class SessionFeature(
     override fun stop() {
         presenter.stop()
     }
+
+    /**
+     * Stops the feature from rendering sessions on the [EngineView] (until explicitly started again)
+     * and releases an already rendering session from the [EngineView].
+     */
+    fun release() {
```

Comment:
We need a new patch in Fenix for calling `sessionFeature.release()` instead of `engineView.release()`, right?

Reply:
Yeah, that will happens as part of https://github.com/mozilla-mobile/fenix/issues/12311

Reply:
nit: We could also add `release()` to the changelog :)

Reply:
Yeah, good point, I keep forgetting about the changelog.

Reply:
> Yeah, that will happens as part of [mozilla-mobile/fenix#12311](https://github.com/mozilla-mobile/fenix/issues/12311)

Awesome :)

Reply:
Filed a follow-up, as discussed: https://github.com/mozilla-mobile/android-components/issues/7657

With a new ID, we should be able to remove the need for the app to call `release`, as the custom tab presenter wouldn't ever render the regular tab.

Reply:
Great, thank you!